sdks/java/core/src/main/java/org/apache/beam/sdk/util/construction/PTransformTranslation.java

```diff
@@ -177,6 +179,9 @@ public class PTransformTranslation {
     checkState(FLATTEN_TRANSFORM_URN.equals(getUrn(StandardPTransforms.Primitives.FLATTEN)));
     checkState(
         GROUP_BY_KEY_TRANSFORM_URN.equals(getUrn(StandardPTransforms.Primitives.GROUP_BY_KEY)));
+    checkState(
```

Comment:
I don't think we need this test, since this transform should never show up in a portable graph. Removing the URN from the proto is a good way to protect us from mistakes, I think.